### PR TITLE
Add Vercel Sponsor Banner

### PR DESCRIPTION
**Pull request by sarthakkundra**
## Description
Added Vercel banner for sponsorship.
Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
#26 
## Type of change
UI
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] This change requires a documentation update.

### CODE REVIEW

**Comment by sarthakkundra** [comment]
Screenshot 2021-07-15 at 1 02 25 AM
@HarshCasper I can see the footer images locally but not on the production deploy. Any idea what could be going wrong?

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -85,7 +85,16 @@ module.exports = {
           ],
         },
         {
-          title: 'News'
+          title: 'Sposnors',
+          items: [
```
Comment (aornugent) [inline website/docusaurus.config.js:89]: "Sponsors"

**Review by aornugent — APPROVED** [review]
Looks great! Hopefully you're able to resolve the footer image.

**Review by HarshCasper — APPROVED** [review]
Just a typo @aornugent mentioned. The rest looks great!

**Comment by sarthakkundra** [comment]
> Just a typo @aornugent mentioned. The rest looks great!

Can you see the Vercel banner in the preview build? I can't see it, but it's visible locally.

**Comment by HarshCasper** [comment]
> > Just a typo @aornugent mentioned. The rest looks great!
> 
> Can you see the Vercel banner in the preview build? I can't see it, but it's visible locally.

Nope. I am not able to see it. It might be an issue with the relative path.

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ module.exports = {
     footer: {
       logo: {
         alt: 'moja global light logo',
-        src: 'static/img/logo-light.png',
+        src: './static/img/logo-light.png',
```
Comment (CIPHERTron) [inline website/docusaurus.config.js:38]: @sarthakkundra I guess changing `src: './static/img/logo-light.png',` to `src: 'img/logo-light.png',` might work!
Reference: https://docusaurus.io/docs/docusaurus.config.js/#optional-fields

Screenshot from 2021-07-17 01-40-33

Comment (sarthakkundra) [reply website/docusaurus.config.js]: Thanks for the reference. Trying this now.

**Pull request merged**